examples/angular/src/app/pages/pages.component.ts

```diff
@@ -124,4 +133,18 @@ export class DotCMSPagesComponent implements OnInit {
      */
     postMessageToEditor({ action: CUSTOMER_ACTIONS.CLIENT_READY, payload: {} });
   }
+
+  #listenToEditorChanges() {
+    this.client.editor.on('changes', (page) => {
+      if (!page) {
+        return;
+      }
+
+      this.$context.update((state) => ({
+        ...state,
+        status: 'loading',
+        error: null,
+      }));
```

Comment:
you have to set the page here right?

Reply:
Yeah, right! Copilot troll me by not adding my clipboard here ahh

Reply:
Good catch